src/main/scala/io/iohk/ethereum/governance/CertificateAuthorityManager.scala

```diff
@@ -10,7 +10,7 @@ import io.iohk.ethereum.vm.utils.{Contract, Utils}
 
 
 trait CertificateAuthorityManager {
-  def isCertificateAuthorityFor(address: Address, slotNumber: BigInt): Boolean
+  def isCertificateAuthorityFor(address: Address, lastBlockHeader: BlockHeader): Boolean
```

Comment:
Minor detail: Maybe we should add a comment regarding the fact that the state at which `lastBlockHeader` leaves the contract is used for executing the query?